cluster/deploy.sh

```diff
@@ -37,6 +37,15 @@ EOF
 # Deploy infra for testing first
 _kubectl create -f ${MANIFESTS_OUT_DIR}/testing
 
+# Deploy CDI with operator.
```

Comment:
unfortunately this has to be added to the new cluster/deploy-operator.sh file too for CI to pass.

We haven't deprecated the non-operator way of deploying kubevirt in our dev environment yet... we should though soon.

Reply:
I think it's already in there

Reply:
So it needs to be in both places?

Reply:
yes.. again sorry about the confusion here. This is on us. We shouldn't have two paths for cluster sync, and won't soon.

Reply:
Okay running locally now, will push new version soon